azclishell/az_completer.py

```diff
@@ -284,15 +296,20 @@ def gen_global_param_completions(self, text):
                         and not text.split()[-1].startswith('--') and \
                         param.startswith('-') and not param.startswith('--') and\
                         self.validate_completion(param, text.split()[-1], text, double=False):
-                    yield Completion(param, -len(text.split()[-1]))
+                    yield Completion(
+                        param, -len(text.split()[-1]),
```

Comment:
The Completion class is a part of a library that I am using. It starts at the cursor position so it has to negative.